stacks/api/passport/policies/apikey.fullaccess.json

```diff
@@ -7,7 +7,7 @@
     {
       "effect": "allow",
       "action": "passport:apikey:*",
-      "resource": "passport:apikey/*",
```

Comment:
Why we needed this change?

Reply:
Because it returns response as 'Forbidden action' when sending any HTTP requests with fullaccess api key which has this resource. And other features like strategy and policy has the same resource as only '*'.